### PR TITLE
Remove some old unused attributes

### PR DESCRIPTION
**Pull request by gtrepta**

### CODE REVIEW

File: kore/src/main/scala/org/kframework/attributes/Att.scala [hunk 1]
```diff
@@ -183,15 +183,13 @@ object Att {
...
   final val CELL_NAME = Key("cellName", KeyType.BuiltIn)
-  final val CIRCULARITY = Key("circularity", KeyType.BuiltIn)
   final val COLOR = Key("color", KeyType.BuiltIn)
   final val COLORS = Key("colors", KeyType.BuiltIn)
   final val COMM = Key("comm", KeyType.BuiltIn)
   final val CONCRETE = Key("concrete", KeyType.BuiltIn)
   final val CONSTRUCTOR = Key("constructor", KeyType.BuiltIn)
   final val CONTEXT = Key("context", KeyType.BuiltIn)
   final val COOL = Key("cool", KeyType.BuiltIn)
-  final val DEPENDS = Key("depends", KeyType.BuiltIn)
```
Comment (Scott-Guest) [inline kore/src/main/scala/org/kframework/attributes/Att.scala]: `circularity` and `depends` should be kept - they're not in use yet but are under development

Comment (ehildenb) [reply kore/src/main/scala/org/kframework/attributes/Att.scala]: Here is an example that uses `circularity`: https://github.com/runtimeverification/pyk/blob/fd06d71322d85f41ecb652a3ac33d438c1bf0e4d/src/tests/integration/k-files/imp-simple-spec.k#L136

And one that uses `depends`: https://github.com/runtimeverification/pyk/blob/fd06d71322d85f41ecb652a3ac33d438c1bf0e4d/src/tests/integration/k-files/imp-simple-spec.k#L141

**Comment by ehildenb** [comment]
Other than keeping `circularity` and `depends`, it looks good. I would ask @dwightguth to also have a look though, maybe some of these attributes mean something for LLVM backend.

**Review by dwightguth — APPROVED** [review]

**Comment by dwightguth** [comment]
Looks good. The test failure is a known transient issue, just rerun the tests.

**Review by Scott-Guest — APPROVED** [review]

**Pull request merged**